src/client/containers/PullRequestDiscussion/PullRequestDiscussion.js

```diff
@@ -31,10 +31,10 @@ export type Props = {
 
 export const getFetchStatus = statusFetchFactory(types.FETCH_PULL_REQUEST_DISCUSSION)
 
-export const getLoggedUser = (state: Object) => state.session.profile.fullName
+export const getLoggedUsername = (state: Object) => state.session.profile.fullName
```

Comment:
If I see correctly, in William's PR the `getPullRequestDiscussion` selector is removed alltogether. Shouldn't we drop my PR then?